Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3 as base
+
+#Docker always runs a root 
+# SO by default /root/.poetry/bin 
+# From get-poetry.py
+# HOME = expanduser("~")
+# POETRY_HOME = os.environ.get("POETRY_HOME") or os.path.join(HOME, ".poetry")
+# POETRY_BIN = os.path.join(POETRY_HOME, "bin")
+# POETRY_ENV = os.path.join(POETRY_HOME, "env")
+# POETRY_LIB = os.path.join(POETRY_HOME, "lib")
+# POETRY_LIB_BACKUP = os.path.join(POETRY_HOME, "lib-backup")
+
+ENV POETRY_HOME=/usr/poetry 
+ENV PATH=$PATH:$POETRY_HOME/bin
```

Comment:
Poetry _should_ just work without these lines, but no problem including them

Reply:
Commented out

Reply:
No poetry will not install without these in the path

Reply:
C:\Google Drive\DevOps\DevOps-Course-Starter [exercise-5 +1 ~5 -0 !]> docker build --target development --tag todo-app:dev .
[+] Building 11.3s (11/11) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                          0.0s
 => => transferring dockerfile: 32B                                                                                                                                           0.0s
 => [internal] load .dockerignore                                                                                                                                             0.0s
 => => transferring context: 35B                                                                                                                                              0.0s
 => [internal] load metadata for docker.io/library/python:3                                                                                                                  11.0s
 => [auth] library/python:pull token for registry-1.docker.io                                                                                                                 0.0s
 => [base 1/5] FROM docker.io/library/python:3@sha256:e8e000f3c551f93d7b03d241e38c1206eb8c8a1f1a6179902f74e068fc98ee59                                                        0.0s
 => [internal] load build context                                                                                                                                             0.0s
 => => transferring context: 3.40kB                                                                                                                                           0.0s
 => CACHED [base 2/5] RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python                                                      0.0s
 => CACHED [base 3/5] WORKDIR /my-work-dir                                                                                                                                    0.0s
 => CACHED [base 4/5] COPY todo_app ./todo_app                                                                                                                                0.0s
 => CACHED [base 5/5] COPY pyproject.toml poetry.toml poetry.lock .env ./                                                                                                     0.0s
 => ERROR [development 1/1] RUN poetry install                                                                                                                                0.2s
------
 > [development 1/1] RUN poetry install:
#11 0.159 /bin/sh: 1: poetry: not found

Reply:
My mistake, apologies, leave them in 🙂 

I was thinking of doing `RUN pip install poetry` in which case you don't need the PATH changes.